### PR TITLE
[WIP] partial fix to 46, corrects centroid calculation for link

### LINKED ISSUE #46: Link to image from status page does not work in OAM browser
Clicking on the `View Image` at the bottom of the upload status page after the upload is finished takes you to a OAM Browser page with an empty/gray map ...somewhere around the South Pole :)

This is an example [`View Image`](http://hotosm.github.io/oam-browser/#/18.1352548305,-72.8718236266,12) link from the completed status page and this is instead [where it should point](https://beta.openaerialmap.org/#/-72.83935546875,18.33366818777604,11/0322112020332/580b07bab0eae7f3b143a8e5?_k=8q0glf) to.

### PR DESCRIPTION
**Pull request by smit1678**
This partially fixes #46. This will fix the link to the correct location on the map, but does not select the correct map grid cell to select the image on the right side bar.

@danielfdsilva Should be good to go pending a quick review from you.

### CODE REVIEW

File: app/assets/scripts/views/status.js [hunk 1]
```diff
@@ -95,19 +96,11 @@ module.exports = React.createClass({
...
-            [bb[3], bb[2]]
-          ]]
-        }
-      };
+      var footprint = image.metadata.footprint;
+      var f = parse(footprint);
+
       var coords = turfCentroid(f).geometry.coordinates;
-      var url = 'http://hotosm.github.io/oam-browser/#/' + coords[0] + ',' + coords[1] + ',12';
+      var url = 'https://beta.openaerialmap.org/#/' + coords[0] + ',' + coords[1] + ',12';
```
Comment (danielfdsilva) [inline app/assets/scripts/views/status.js]: It would be better to store this url in the config file.

**Comment by smit1678** [comment]
@danielfdsilva Made some changes based on your suggestions. Give this a review for cleanliness. One thing to note is that we can't add the image ID into the URL since it is not available within the metadata. Image ID is only added once the image is indexed by the Catalog. Current method does not post metadata to the catalog, only waits until the scheduled task takes place (every 10 mins).

File: app/assets/scripts/views/status.js [hunk 2]
```diff
@@ -15,6 +16,26 @@ function dateFormat (date) {
...
+// Use image metadata to construct OAM Browser URL describing the map view,
+// associated grid tile. Image ID is not available since it has not been indexed,
+// by the Catalog yet
+// adapted from "https://github.com/hotosm/openaerialmap.org/blob/master/app/assets/scripts/main.js#L36-L50
+const constructUrl = (imgData) => {
+  const previewZoom = 10;
+  // Use turf to calculate the center of the image
+  const footprint = parse(imgData.metadata.footprint);
+  const center = turfCentroid(footprint).geometry.coordinates;
+  console.log(imgData);
```
Comment (danielfdsilva) [inline app/assets/scripts/views/status.js]: Can we remove the log?

File: app/assets/scripts/views/status.js [hunk 3]
```diff
@@ -15,6 +16,26 @@ function dateFormat (date) {
...
+
+  // Calculate the tile quadkey for the image using Mapbox tilebelt
+  // * a square at zoom Z is the same as a map tile at zoom Z+3 (previewZoom)
+  const tile = tilebelt.pointToTile(center[0], center[1], previewZoom + 3);
+  const quadKey = tilebelt.tileToQuadkey(tile);
+  const mapView = center[0] + ',' + center[1] + ',' + previewZoom;
+  // Return OAM Browser URL including map view, tile, and image id
+  return `https://beta.openaerialmap.org/#/${mapView}/${quadKey}/`;
```
Comment (danielfdsilva) [inline app/assets/scripts/views/status.js]: Maybe move this to the config

File: app/assets/scripts/views/status.js (hunk 2 above)
Comment (smit1678) [reply app/assets/scripts/views/status.js]: :bowtie:

**Comment by smit1678** [comment]
To keep this fresh, still a few details and clean up to do. Still a work in progress.

File: app/assets/scripts/views/status.js (hunk 3 above)
Comment (smit1678) [reply app/assets/scripts/views/status.js]: Done

**Comment by smit1678** [comment]
@danielfdsilva Fixed the console and moved the browser url to the config file. I think this is good to go unless you have some additional recommendations.

**Comment by danielfdsilva** [comment]
@smit1678 :shipit:

**Pull request merged**